packages/react-server-test-pages/package.json

```diff
@@ -19,5 +20,8 @@
     "react-server-cli": "^0.2.6",
     "react-server-data-bundle-cache": "^0.2.6",
     "superagent": "1.2.0"
+  },
+  "devDependencies": {
+    "node-inspector": ""
```

Comment:
Yea tbh I'm not sure which to pick! The latest stable version that I used for getting this to work is 0.12.8 - I wasn't sure if I should pin to that specific version or accept patches (~0.12.8?), etc. Any guidance?